add systemd timers

rphillips opened the pull request.

pronoiac commented:
I'm unfamiliar with systemd. Checking this: 
* I *think* tarsnap-archive & tarsnap-prune have been copied to /usr/local/bin, without the .sh extension. If so, I'd add a note about this. 
* to be explicit, the `start` and `enable` lines have to be done repeatedly, for `prune` and each frequency, right? 
* what distro did you build and test on? 

I'm not trying to idiot-proof it, but I do want to make it as easy to use as I can manage.

rphillips commented:
Hey James, I cleaned this up and added a bit more documentation. Arch Linux (and possibly other distros) are not including a cron by default now.

I followed this [documentation](https://wiki.archlinux.org/index.php/Systemd/Timers).

pronoiac commented:
Thanks!
